src/node/nodestate.h

```diff
@@ -527,7 +527,7 @@ namespace ccf
       initiate_join(args);
 
       using namespace std::chrono_literals;
-      join_timer = timers.new_timer(1s, [this, args]() {
+      join_timer = timers.new_timer(2s, [this, args]() {
```

Comment:
Relaxing this timer here. Otherwise, since quote verification takes ~2secs today, a joining node may DOS the leader by issuing join requests faster than the leader can process it.